po/de.po

```diff
@@ -0,0 +1,1112 @@
+"Passwort kann nicht wiederhergestellt werden. Bitte sicherheitshalber "
+"notieren"
+
+#: src/js/services/profileService.js
+msgid "Password needed"
+msgstr "Passwort benötigt"
+
+#: src/js/controllers/password.js
+msgid "Passwords do not match"
+msgstr "Passwörter stimmen nicht überein"
+
+#: public/views/join.html
+msgid "Paste invitation here"
+msgstr "Einladung hier einfügen"
+
+#: public/views/import.html
+msgid "Paste the backup plain text code"
+msgstr "Den Klartext des Backups einfügen"
+
+#: public/views/modals/paypro.html
+msgid "Pay To"
+msgstr "Zahle an"
+
+#: public/views/modals/tx-status.html
+msgid "Payment Accepted"
+msgstr "Zahlung angenommen"
+
+#: public/views/modals/txp-details.html
+msgid "Payment Proposal"
+msgstr "Zahlungsvorschlag"
+
+#: public/views/modals/tx-status.html
+msgid "Payment Proposal Created"
+msgstr "Zahlungsvorschlag erstellt"
+
+#: src/js/services/notificationsService.js
+msgid "Payment Proposal Rejected"
+msgstr "Zahlungsvorschlag abgelehnt"
+
+#: src/js/services/notificationsService.js
+msgid "Payment Proposal Rejected by Copayer"
+msgstr "Zahlungsvorschlag wurde vom Copayer abgelehnt"
+
+#: src/js/services/notificationsService.js
+msgid "Payment Proposal Signed by Copayer"
+msgstr "Zahlungsvorschlag wurde vom Copayer abgezeichnet"
+
+#: public/views/walletHome.html
+msgid "Payment Proposals"
+msgstr "Zahlungsvorschläge"
+
+#: src/js/controllers/walletHome.js
+msgid "Payment Protocol not supported on Chrome App"
+msgstr "Zahlungsprotokoll wird nicht von der Chrome App unterstützt"
+
+#: public/views/modals/tx-status.html
+msgid "Payment Rejected"
+msgstr "Zahlung abgelehnt"
+
+#: public/views/modals/tx-status.html src/js/services/notificationsService.js
+msgid "Payment Sent"
+msgstr "Zahlung gesendet"
+
+#: public/views/modals/txp-details.html
+msgid "Payment accepted..."
+msgstr "Zahlung angenommen..."
+
+#: public/views/modals/txp-details.html
+msgid "Payment details"
+msgstr "Zahlungsdetails"
+
+#: public/views/modals/txp-details.html
+msgid "Payment finally rejected"
+msgstr "Zahlung endgültig abgelehnt"
+
+#: public/views/modals/paypro.html
+msgid "Payment request"
+msgstr "Zahlungsanforderung"
+
+#: public/views/modals/txp-details.html
+msgid "Payment sent!"
+msgstr "Zahlung gesendet!"
+
+#: public/views/walletHome.html
+msgid "Payment to"
+msgstr "Zahlung an"
+
+#: public/views/preferencesDeleteWallet.html
+msgid "Permanently delete this wallet. THIS ACTION CANNOT BE REVERSED"
+msgstr ""
+"Wallet dauerhaft löschen. DIESE AKTION KANN NICHT RÜCKGÄNGIG GEMACHT WERDEN"
+
+#: public/views/create.html
+msgid "Personal Wallet"
+msgstr "Persönliches Wallet"
+
+#: src/js/controllers/create.js src/js/controllers/join.js
+msgid "Please enter the required fields"
+msgstr "Bitte die benötigten Felder ausfüllen"
+
+#: src/js/controllers/import.js
+msgid "Please, select your backup file"
+msgstr "Bitte die Sicherungsdatei wählen"
+
+#: src/js/controllers/index.js
+msgid "Portuguese"
+msgstr "Portuguese"
```

Comment:
if you intended all language names to be represented in the target language, then i think this should be "português"